qutebrowser/misc/sql.py

```diff
@@ -483,7 +483,7 @@ def delete(self, field: str, value: Any) -> None:
         q = self.database.query(f"DELETE FROM {self._name} where {field} = :val")
         q.run(val=value)
         if not q.rows_affected():
-            raise KeyError('No row with {field} = "{value}"')
+            raise KeyError(f'No row with {field} = "{value}"')
```

Comment:
I know this is a bot, but it probably makes more sense to also use `{value!r}` instead of `"{value}"` here

Reply:
I'm half bot half human.

Our bot detected the problem, then a real developer triaged the issue and approved the bot to create the issue and PR. 

Now it's back in the hands of the humans monitoring the PRs :)

I'll make that change now

Reply:
> I'll make that change now

You didn't remove the quotes.